README.md

```diff
@@ -1,3 +1,59 @@
 # Castle Crashers
 
-A CMPT-306 game made by a team of 6 people in the span of 1.5ish months!
\ No newline at end of file
+A CMPT-306 dungeon crawler game built in Unity and C# by a team of 7 people in the span of 1.5ish months!
+The name was a coincidence; we just like the ring to it (so did The Behemoth for the real Castle Crashers!)
+Note: the class focused on Game Mechanics, most of the visual and audio assets were used from free libraries or purchased from the Unity Store.
+
+## Overview
+The goal of the game is to traverse deeper and deeper to new levels of the castle, wield a variety of weapons to battle a variety of enemies, utilize gathered loot and coins to get stronger via opening chests, recieving permanent power upgrades, and buying stat & skill upgrades. Once the player reach the end of their journey, usually ended from a gruesome death, they would recieve a score based on their performance in the game.
```

Comment:
- "recieving" should be "receiving"
- I would word "usually ended from a gruesome death" as "usually caused by a gruesome death"
- "Once the player reach the end of their journey" -> change "reach" to "reaches"
- "they would recieve" ->
  - change "recieve" to "receive"
  - I'd get rid of "would"